ERCS/erc-7649.md

```diff
@@ -0,0 +1,446 @@
+
+    - **Liquidity Management**: The bonding curve contract's role in managing liquidity through the bonding curve
+        mechanism is essential. This design choice facilitates a dynamic and automated pricing model based on supply
+        and demand, contributing to the overall liquidity and tradability of NFT shares.
+
+2.  **Liquidity Mechanisms**:
+    - **Versatility**: The standard's allowance for various liquidity mechanisms, including bonding curves, liquidity
+        locking with ERC-20 tokens, or native ETH liquidity pools, reflects a commitment to versatility.
+        This flexibility enables developers to tailor the liquidity mechanism based on the specific requirements of
+        their NFT projects, promoting innovation and diversity within the ecosystem.
+
+    - **Adaptability**: By providing flexibility in choosing the liquidity mechanism, the standard ensures that
+        developers can adapt to evolving market trends and user preferences. This adaptability is crucial for the
+        long-term sustainability and relevance of NFT projects.
+
+3. **Bonding Curve Mechanism**:
+    - **Dynamic Pricing**: The adoption of a bonding curve mechanism to determine the price of Tradable Shares aligns
+        with the goal of encouraging liquidity providers to engage in buying and selling NFT shares.
+        The dynamic pricing, influenced by the curve's formula, ensures that the market for Tradable Shares remains
+        responsive to changing conditions.
+
+    - **Customization for Creators**: The decision to allow creators to customize bonding curve parameters, such as
+        slope and intercept, empowers them to tailor the liquidity mechanism to the unique needs and characteristics of
+        their projects. This customization fosters creativity and innovation within the NFT space.
+
+4. **Guaranteed Creative Fees**:
+    - **Creator Incentives**: The emphasis on guaranteeing creative fees for NFT creators is foundational to sustaining
+        a thriving ecosystem. By enabling creators to specify and receive a percentage of transaction fees, the standard
+        aligns incentives and rewards creators for their contributions, fostering a sustainable and creator-friendly
+        environment.
+
+5. **Implementation Independence**:
+    - **Developer Freedom**: The standard's implementation-agnostic approach is motivated by the desire to provide
+        developers with the freedom to choose and design the most suitable liquidity mechanism for their NFT projects.
+        Whether interacting with ERC-20 tokens or native ETH, this independence ensures that developers can make
+        informed choices based on the specific requirements of their projects.
+
+The rationale behind these design choices is to create a Tradable Shares standard that is not only technically sound but
+also flexible, adaptable, and supportive of diverse and creative implementations within the ERC-721 ecosystem.
```

Comment:
Your rationale section is very... generic. It doesn't really give information about any specifics in your proposal. Normally the rationale section should explain individual technical choices made within the proposal itself.

A poor example for your proposal would be explaining why you chose 18 decimals for the fee percentage. As the author, I'm sure you can come up with better things to explain :rofl:

Reply:
Updated both Specification and Rationale sections. Made Specification section more strict, removed the support of any other liquidity mechanism, except for bonding curves – focus solely on the bonding curves.

Every item is Rationale corresponds to the item in Specification and explains "why", while items in Specification explain "what".